File: python/S1_BASIC/L4_BASICMATHS/reversenum.py
class Solution:
    def __init__(self,num):
        self.num=num
    def reverse(self):
        r=0
        while self.num>0:
            r=r*10+(self.num%10)
            self.num//=10
        return r
    
if __name__=="__main__":
    num=int(input("enter number :"))
    s=Solution(num)
    print(s.reverse())

File: JAVA/S1_BASIC/L4_BASICMATHS/reversenum.java
class solution{
    int  reverse(int num){
        int r=0;
        while(num>0){
            r=r*10+num%10;
            num/=10;
        }
        return r;
    }
}
public class reversenum{
    public static void main(String[] args){
    solution s=new solution();
    System.out.println(s.reverse(3562));
    }

}